src/scss/_override-variables.scss

```diff
@@ -101,15 +101,21 @@ $vl-icon-font-location: '@govflanders/vl-ui-design-system-style/assets/font/icon
 }
 
 // vl-tabs
+.vl-tabs__wrapper {
+  width: fit-content;
+  margin: 0 auto;
```

Comment:
dit breekt nog de layout bij kleinere screensize zie ik. Is het eigenlijk wel nodig om van de huisstijl af te wijken en dit menu te centreren? Full width ziet het er ook goed uit

Reply:
bij toggelen van menu op de algemeen tab:
![image](https://github.com/OnroerendErfgoed/vue_component_library/assets/62388265/c5cad5a9-63f1-4130-a151-2328783f703b)

Reply:
nvm dat is op te lossen in plannen zelf.